app/Main.hs

```diff
@@ -70,6 +70,8 @@ defaultProject =
           , projectGenerateOnly = False
           , projectForceReload = False
           , projectPkgConfigFlags = []
+          , projectCModules = []
+          , projectLoadStack = []
```

Comment:
What's the purpose of this?

Reply:
`projectCModules` is the list of modules to be included in the compilation command.
`projectLoadStack` is the list of `.carp` files being loaded so far. It's used in `Dynamic.carp:current-file` to get the file currently being loaded.